elements/pfe-sass/extends/_extends.scss

```diff
@@ -56,3 +56,84 @@
     color: pfe-fetch(text) !important;
   }
 }
+
+%ordered-list-primary {
+  // size of the font inside of the circle
+  --FontSize: var(--pfe-list-FontSize, 24px);
```

Comment:
Can we incorporate some kind of prefixing here to prevent accidental conflicts?  `pfe-ol` maybe?

Reply:
Yeah good point.  This definitely has to be more specific.

Reply:
I added a pretty specific prefix: `pfe-ol-list--primary` since this design is so different from the rest of the pfe-list. 🤷‍♀️